youtube_dl/extractor/vlive.py

```diff
@@ -261,3 +265,62 @@ def _real_extract(self, url):
 
         return self.playlist_result(
             entries, channel_code, channel_name)
+
+
+class VLivePlaylistIE(InfoExtractor):
+    IE_NAME = 'vlive:playlist'
+    _VALID_URL = r'https?://(?:(?:www|m)\.)?vlive\.tv/video/(?P<video_id>[0-9]+)/playlist/(?P<id>[0-9]+)'
+    _TESTS = [{
+        'url': 'http://www.vlive.tv/video/30824/playlist/30826',
+        'info_dict': {
+            'id': '30826',
+            'title': 'TWICE TV5 - TWICE in SWITZERLAND'
+        },
+        'playlist_mincount': 20
+    }, {
+        'url': 'http://www.vlive.tv/video/22867/playlist/22912',
```

Comment:
If I make it `only_matching` does that mean it will only test that the URL matches the regex? If so, that defeats the point of that test. The URL in the second test links to a video which appears to be linked to the playlist somewhat differently to the URL in the first test.

From my observations, if someone were to download the webpage using only the video id from the given URL, it would not link to the playlist, whereas it would with the URL in the first test. Therefore I think both tests should perform the full webpage download.

i.e:
`http://www.vlive.tv/video/30824` the HTML for this page does contain the playlist
`http://www.vlive.tv/video/22867` the HTML for this page does not

I can add a code comment to clarify this if my assumption about `only_matching` is correct.